src/ontology/mondo-ingest.Makefile

```diff
@@ -160,15 +160,17 @@ mappings: sssom $(ALL_MAPPINGS)
 #################
 # Utils #########
 #################
-# Documentation for this commands in this section is in: `docs/developer/ordo.md`
-
+# Documentation for `repo   rt-mapping-annotations` and `update-jinja-sparql-queries`: `docs/developer/ordo.md`
 report-mapping-annotations:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/report_mapping_annotations.py
 
 update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+config/%_term_exclusions.txt: config/%_exclusions.tsv component-download-%.owl $(TMPDIR)/%_relevant_signature.txt
```

Comment:
@matentzn Only able to get success on the mirror and component recipes for: ICD10CM, ICD10WHO, and OMIM. I saved the error messages for the other ones, but I haven't had time to look much into them yet. I know that for ORDO, I got a 404 not found.

Reply:
This will deal with the ORDO issue: https://github.com/monarch-initiative/mondo-ingest/pull/41 (merge, then run `sh run.sh make update_repo`)

Reply:
Identical with reports/component_signature-omim.tsv

Reply:
Not quite identical, but same purpose.

Reply:
Merged. Will run that now!

Full error messages for reference:
### DO
```
rget `mondo-ingest-full.owl'
Makefile:516: warning: ignoring old commands for target `mondo-ingest-full.owl'
if [ true  = true ] && [ true  = true ]; then robot --catalog catalog-v001.xml merge -I http://purl.obolibrary.org/obo/doid.owl \
	annotate --ontology-iri http://purl.obolibrary.org/obo/mondo-ingest/component-download-doid.owl annotate -V http://purl.obolibrary.org/obo/mondo-ingest/releases/2022-07-21/component-download-doid.owl --annotation owl:versionInfo 2022-07-21 -o tmp/component-download-doid.owl.owl; fi
if [ true  = true ]; then robot --catalog catalog-v001.xml query -i "tmp/component-download-doid.owl.owl" -q "../sparql/doid-relevant-signature.sparql" tmp/doid_relevant_signature.txt; fi
QUERY PARSE ERROR query cannot be parsed:
Line 7, column 12: Unresolved prefixed name: rdfs:subClassOf
For details see: http://robot.obolibrary.org/query#query-parse-error
Use the -vvv option to show the stack trace.
Use the --help option to see usage information.
make: *** [tmp/doid_relevant_signature.txt] Error 1
```

### ORDO
Got a 404:
```
/mondo-ingest/component-download-ordo.owl annotate -V http://purl.obolibrary.org/obo/mondo-ingest/releases/2022-07-21/component-download-ordo.owl --annotation owl:versionInfo 2022-07-21 -o tmp/component-download-ordo.owl.owl; fi
org.semanticweb.owlapi.io.OWLOntologyCreationIOException: http://www.orphadata.org/data/ORDO/ORDO_en_4.0.owl
Use the -vvv option to show the stack trace.
Use the --help option to see usage information.
make: *** [component-download-ordo.owl] Error 1
```

### NCIT
(i think i need to run again; some of my terminal history got erased for some reason)
[ncit-errors.txt](https://github.com/monarch-initiative/mondo-ingest/files/9169251/ncit-errors.txt)

Reply:
These are all resolved.

Indeed, the NCIT Java heap space issue still remains. I will make a comment specifically about that.